optimizer_wrapper.rb

```diff
@@ -400,6 +400,21 @@ def self.split_independent_vrp(vrp)
     # - there is multimodal subtours
     return [vrp] if (vrp.vehicles.size <= 1) || vrp.services.empty? || vrp.subtours&.any? # there might be zero services
 
+    # Accumulate the skills of services in a relation of ALL_OR_NONE_RELATIONS type
+    vrp.relations.each{ |relation|
+      next unless Models::Relation::ALL_OR_NONE_RELATIONS.include?(relation.type)
+
+      if Models::Relation::ON_SERVICES_TYPES.include?(relation.type)
```

Comment:
Why is this a different condition than above ?

Reply:
We might add new "all_or_none" type relations which is on "vehicles" in the future and this makes sure that we only apply the skill accumulation to relations related to services and get alerted about the need for human intevention before the dev hits the servers.